src/components/types.ts

```diff
@@ -32,7 +32,8 @@ export interface InputProps extends InputStatus, InputHTMLAttributes<HTMLInputEl
    */
   endIcon?: ReactNode;
 
-  weight?: number;
+  fontWeight?: number;
+  fontSize?: string;
```

Comment:
Do we have full freedom of these strings and number? Shall we use a set of options instead?

Reply:
good shout - i've added in custom types for FontWeights and FontSizes now :)